src/main/java/Duke.java

```diff
@@ -6,5 +8,63 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("____________________________________________________________");
```

Comment:
Perhaps could use the refactoring, to extract this section of code and put it in an introductoryMessage function